README.md

```diff
@@ -80,21 +80,22 @@ spec:
   replicas: 1
   server:
     distribution:
-      name: ollama
+      name: starter
     containerSpec:
-      port: 8321
       env:
       - name: INFERENCE_MODEL
-        value: "llama3.2:1b"
+        value: "llama3.2:3b"
       - name: OLLAMA_URL
         value: "http://ollama-server-service.ollama-dist.svc.cluster.local:11434"
+      - name: ENABLE_OLLAMA
+        value: ollama
```

Comment:
I find this value for the name a bit odd... :smile:   but I guess is outside of scope here